src/App.css

```diff
@@ -1453,3 +1453,104 @@ a.button-dim {
 .rotate180 {
   transform: rotate(180deg);
 }
+
+/* Custom checkbox style starts */
+.checkbox-container {
+  display: block;
+  position: relative;
+  padding-left: 2.2em;
+  user-select: none;
+}
+
+.checkbox-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkbox {
+  position: absolute;
+  top: 0.1em;
+  left: 0.1em;
+  height: 1em;
+  width: 1em;
+  background-color: #fff;
+}
+
+.checkbox-container input ~ .checkbox,
+.checkbox-container:hover input ~ .checkbox,
+.checkbox-container input:checked ~ .checkbox {
+  background-color: transparent;
+  border: 0.1em solid #fff;
+}
+
+/* Create the checkbox/indicator (hidden when not checked) */
+.checkbox:after {
+  content: '';
+  position: absolute;
+  display: none;
+}
+
+/* Show the checkbox when checked */
+.checkbox-container input:checked ~ .checkbox:after {
+  display: block;
+}
+
+/* checkbox indicator */
+.checkbox-container .checkbox:after {
+  left: 0.3em;
+  bottom: 0.2em;
+  width: 0.3em;
+  height: 0.65em;
+  border: solid white;
+  border-width: 0 0.15em 0.15em 0;
+  transform: rotate(45deg);
+}
+/* Custom checkbox style ends */
+
+.gift-code-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-self: start;
+  margin: 3vh auto;
+  align-items: center;
+  width: 100%;
+}
+
+.gift-code-wrapper svg {
+  cursor: default;
+}
+
+.gift-code-wrapper input {
+  margin: 0 10px;
+  min-width: 180px;
+}
+
+@media only screen and (min-width: 767px) {
+  .gift-code-wrapper {
+    width: 80%;
+  }
+
+  .gift-code-wrapper input {
+    min-width: 300px;
+  }
+}
+
+.copy-icon-wrapper {
+  display: 'inline-flex';
+}
+
+.copy-icon-wrapper svg {
+  cursor: pointer;
+}
```

Comment:
Were these checkbox, gift code, and copy icon styles unintentionally added? I don't see how they relate to the pull request, so I assume they were accidentally included when resolving a merge conflict (or something similar).

Reply:
Yes, I believe so they got added during merge conflict during a rebase. I will remove them.